modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -2,17 +2,17 @@
 :description: Manage the throughput of Kafka traffic with configurable properties.
 :page-categories: Management, Networking
 
-Manage the throughput of Kafka traffic at the cluster level, with configurable properties that limit and protect the use of disk and network resources for individual brokers and for an entire cluster. Set broker-wide throughput limits for Kafka API traffic.
+Redpanda allows cluster administrators to configure throughput limits at both the broker level and the client level. For brokers, these configurable limits help protect the use of disk and network resources. Limits applied on the broker apply to all clients that produce or consume there. Alternatively, the client limits allow you to selectively throttle a single client or group of clients. These limits help protect individual clients from 'noisy neighbors' that may consume extreme amounts of available throughput and crowd out other clients.
 
 == Broker-wide throughput limits
 
 The network bandwidth and disk utilization of brokers may be overloaded by clients that produce or consume throughput without limits. To prevent resource overloading caused by unconstrained throughput and to configure back pressure, Redpanda provides runtime-configurable properties that limit and balance throughput of Kafka API traffic.
```

Comment:
'network & storage bandwidth of  brokers may be overloaded by. '

Reply:
Modified.